app/views/job_invocations/_form.html.erb

```diff
@@ -49,7 +49,11 @@
                   <%= job_template_fields.fields_for :input_values do |input_values_fields| %>
                     <% job_template.template_inputs.where(:input_type => 'user').each do |input| %>
                       <%= input_values_fields.fields_for input.id.to_s, @composer.template_invocation_input_value_for(input) do |input_fields| %>
-                        <%= text_f input_fields, :value, :label => input.name, :help_inline => input.description, :required => input.required %>
+                        <% unless input.options.blank? %>
+                          <%= selectable_f input_fields, :value, input.options_array, {:include_blank => !input.required }, :require => input.required, :label => input.name, :help_inline => input.description %>
+                        <% else %>
+                          <%= textarea_f input_fields, :value, :label => input.name, :help_inline => input.description, :required => input.required, :rows => 2 %>
```

Comment:
this renders big fields even for things like numbers, how about row => 1, user can enlarge in browser if it's needed